Adapt wording around JSON Logs Preprocessing in Pipelines (#8596)
DataDog/documentation: content/en/logs/processing/_index.md

For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.

> ## JSON Logs Preprocessing

Reviewer: ```suggestion
## JSON Logs Pre processing
```

> JSON Logs preprocessing applies on all logs before they actually enter [Log Pipelines][1] processing. Preprocessing runs a series of operation based on reserved attributes:

Reviewer: ```suggestion
JSON Logs preprocessing applies on all logs before they actually enter [Log Pipelines][1] processing. Preprocessing runs a series of operations based on reserved attributes:
```

> JSON Logs preprocessing applies on all logs before they actually enter [Log Pipelines][1] processing. Preprocessing runs a series of operation based on reserved attributes:

Reviewer: ```suggestion
JSON Logs pre processing applies on all logs before they actually enter [Log Pipelines][1] processing. Pre processing runs a series of operation based on reserved attributes:
```

* Trigger new [log integrations][16] based on the **source** of incoming logs.
* Append incoming logs with all [**host** tags][22].

> * Apply Reserved Attribute Remappers Processors (namely [**Date** Remapper][12], [**Status** Remapper][15], [**Service** Remapper][19], [**Message** Remapper][20] and [**Trace ID** Remapper][21]) for the related JSON attributes of all incoming JSON logs.

Reviewer: ```suggestion
* Apply reserved attribute remapper processors (namely [**date** remapper][12], [**status** remapper][15], [**service** remapper][19], [**message** remapper][20] and [**trace ID** remapper][21]) for the related JSON attributes of all incoming JSON logs.
```

> JSON Logs preprocessing comes with a default configuration that work for standard log forwarders. Edit this configuration at any time to adapt to custom or specifc log forwarding approaches. To change the default values, go to the [Configuration page][5] and edit the `JSON Logs Preprocessing`:

Reviewer: ```suggestion
JSON Logs pre processing comes with a default configuration that works for standard log forwarders. Edit this configuration at any time to adapt to custom or specific log forwarding approaches. To change the default values, go to the [configuration page][5] and edit `Pre processing for JSON logs`:
```

For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.

## JSON Logs Preprocessing

JSON Logs preprocessing comes with a default configuration that work for standard log forwarders. Edit this configuration at any time to adapt to custom or specifc log forwarding approaches. To change the default values, go to the [Configuration page][5] and edit the `JSON Logs Preprocessing`:

> {{< img src="logs/processing/json_logs_preprocessing.gif" alt="JSON Logs Preprocessing Tile"  style="width:80%;">}}

Author: I don't get why this one is broken in preview (although all CI tests passed).

+ The alternative text is still Reserved Attribute Tile 🤔 , which is the alternative text for the image we had in the previous version.
